src/libraries/Native/Unix/System.Security.Cryptography.Native.Android/pal_eckey.h

```diff
@@ -0,0 +1,63 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#pragma once
```

Comment:
Is this used in other places in the codebase? It is technically non-standard even though many compilers support it. We should confirm we are permitted to use this in this area.

Reply:
A lot of the files in `src/libraries/Native` currently use `#pragma once`, including the ones already in `System.Security.Cryptography.Native.Android`